internal/repository/prisma/step_run.go

```diff
@@ -381,6 +381,20 @@ var genericRetry = func(l *zerolog.Event, maxRetries int, f func() error, msg st
 	return nil
 }
 
+func (s *stepRunEngineRepository) ReleaseWorkerSemaphoreSlot(ctx context.Context, tenantId, stepRunId string) (*dbsqlc.WorkerSemaphoreSlot, error) {
```

Comment:
This method seems to be unused? And seems like a duplicate of `ReleaseStepRunSemaphore(ctx context.Context, tenantId, stepRunId string) error` which is used?